manifests/python.yml

```diff
@@ -93,29 +93,23 @@ tests/:
           Test_InsecureAuthProtocol_ExtendedLocation: missing_feature
           Test_InsecureAuthProtocol_StackTrace: missing_feature
         test_insecure_cookie.py:
-          TestInsecureCookie:
-            '*': v1.19.0
-            fastapi: v2.16.0-dev
+          TestInsecureCookie: v3.1.0.dev (v1.19.0 or fastapi v2.16.0, but without telemetry metrics)
           TestInsecureCookieNameFilter: missing_feature
-          TestInsecureCookie_ExtendedLocation: missing_feature
+          TestInsecureCookie_ExtendedLocation: v3.1.0.dev
           TestInsecureCookie_StackTrace: missing_feature
         test_ldap_injection.py:
           TestLDAPInjection: missing_feature
           TestLDAPInjection_ExtendedLocation: missing_feature
           TestLDAPInjection_StackTrace: missing_feature
         test_no_httponly_cookie.py:
-          TestNoHttponlyCookie:
-            '*': v1.19.0
-            fastapi: v2.16.0-dev
+          TestNoHttponlyCookie: v3.1.0.dev (v1.19.0 or fastapi v2.16.0, but without telemetry metrics)
```

Comment:
To keep the manifest more informative about minimum versions, maybe we can change this:
```
@missing_feature(library="python", reason="Metrics not implemented")
```
to a version check in the decorator, and not touching the manifest?

Reply:
But then, we don't see we lack the telemetry support in the manifest in older version. 
So isn't is worse?

Reply:
done